nio_cli/cli.py

```diff
@@ -23,8 +23,8 @@
   nio --version
 
 Options:
-  -p PORT --port=PORT               Specify nio port [default: 8181].
-  -i IP --ip=IP                     Specify nio ip address [default: 127.0.0.1].
+  -p PORT --port=PORT               Specify nio port.
```

Comment:
why no defaults?

Reply:
This was done because docopt would hold a --port value of `8181` and a --ip value of `0.0.0.0` for every command. So if I ran `nio config --port 8282 --ip 127.0.0.1` these values would be changed in nio.conf but if I later ran`nio config --pubkeeper-hostname host --pubkeeper-token token` my port and ip values would get changed back to 8181, 0.0.0.0 even though I didn't specify these values.

To keep the behavior of the defaults, I added this change to the base command class here:
https://github.com/niolabs/nio-cli/blob/fd8036fec0f931a0f8812a41f9f8951fe5763c73/nio_cli/commands/base.py#L10-L15
So that all commands that interact with the CLI fallback to a default host/port.

I would have kept the default values in docopt but then I would have had to do validation on the project config to see if host/port was being changed from or to the default values and the logic seemed more fuzzy that way.